articles/matlab-image-database/index.md

```diff
@@ -0,0 +1,120 @@
+### How to create face database using matlab.
+### Introduction.
+Face databases are imagery data that are used for testing face processing algorithms. In the content of biometrics, databases are collected and used to evaluate the performance of the face recognition biometrics system.
+There are so many image databases available on the internet but if you want to make a database of your faces then you can use this program. The framework of this program can be used to create the database for other many objects.
+In this case, we will use the WEBCAM to make the number of required faces. Face database can be used to implement projects such as face recognition.
```

Comment:
`WEBCAM to make the number of required faces`

Could you explain this sentence clearly? It looks unclear